src/sysfiles.h

```diff
@@ -114,7 +114,7 @@ void SyBufSetEOF(Int fid);
 **  Right now GAP does not read nonascii files, but if this changes sometimes
 **  'SyFopen' must adjust the mode argument to open the file in binary mode.
 */
```

Comment:
Just to take it match up with what's now in `sysfiles.c`:
```suggestion
**
**  If <transcompress> is TRUE, files with names ending '.gz' will be
**  automatically compressed/decompressed using gzip.
*/
```

Reply:
thanks (done in another rearrange)